contribute/design/cstor-operator/day2ops.md

```diff
@@ -0,0 +1,309 @@
+### Introduction
+
+This document focuses on cStor day 2 operations work flow via the proposed specs
+of CSPC and CSP mentioned in contribute/design/cstor-operator/specs.md
+
+### Pool Expansion
+
+* “As a OpenEBS user, I should be able to add block devices to CSPC to increase
+   the pool capacity.” *
+
+Steps To Be Performed By User:
+
+1. `kubectl edit cspc your_cspc_name`
+
+2. Put the block devices in the CSPC spec against the correct kubernetes nodes.
+
+
+**Consider following example for stripe pool expansion**
+
+Current CSPC is following:
+
+
+This CSPC corresponds to a stripe pool on node `gke-cstor-it-default-pool-1`
+
+```
+
+apiVersion: openebs.io/v1alpha1
+kind: CStorPoolCluster
+metadata:
+  name: cstor-pool-stripe
+spec:
+  type: stripe
+  nodes:
+
+  - hostName: gke-cstor-it-default-pool-1
+    groups:
```

Comment:
should we be specific again here : `groups` -> `raidGroups` or `raidConfig`:

```
  nodes:
    - raidConfig:
        - type: stripe
          blockdevices: 
          - bd-....
          - bd-...
```

While the above works for single group, how do I specify raid10, raid50...

Can raid10 be specified as:

```
  nodes:
    - raidConfig:
        - type: mirror
          blockdevices: 
          - bd-....
          - bd-...
        - type: mirror
          blockdevices: 
          - bd-....
          - bd-...
```